pkg/utils/secrets/basic_auth.go

```diff
@@ -86,17 +78,9 @@ func (s *BasicAuthSecretConfig) Generate() (DataInterface, error) {
 func (b *BasicAuth) SecretData() map[string][]byte {
 	data := map[string][]byte{}
 
-	switch b.Format {
```

Comment:
I would appreciated if you add a comment on the Format field for BasicAuthSecretConfig struct on why it cannot and should not be removed.
Can we remove the Format field from the BasicAuth struct?